react-day-picker.d.ts

```diff
@@ -0,0 +1,171 @@
+
+    interface Modifiers {
+        today: Modifier | Modifier[];
+        outside: Modifier | Modifier[];
+        [other: string]: Modifier | Modifier[] | undefined;
+    }
+
+    interface DayModifiers {
+        today: boolean | undefined;
+        outside: boolean | undefined;
+        [other: string]: boolean | undefined;
+    }
+
+    interface Props {
+        canChangeMonth?: boolean;
+        captionElement?: React.ReactElement<Partial<CaptionElementProps>> |
+            React.ComponentClass<CaptionElementProps> |
+            React.SFC<CaptionElementProps>;
+        className?: string;
+        classNames?: ClassNames;
+        containerProps?: React.HTMLAttributes<HTMLDivElement>;
+        disabledDays?: Modifier | Modifier[];
+        enableOutsideDays?: boolean;
+        firstDayOfWeek?: number;
+        fixedWeeks?: boolean;
+        fromMonth?: Date;
+        initialMonth?: Date;
+        labels?: { previousMonth: string; nextMonth: string; };
+        locale?: string;
+        localeUtils?: LocaleUtils;
+        modifiers?: Partial<Modifiers>;
+        month?: Date;
+        months?: [string, string, string, string, string, string, string, string, string, string, string, string];
+        navbarElement?: React.ReactElement<Partial<NavbarElementProps>> |
+            React.ComponentClass<NavbarElementProps> |
+            React.SFC<NavbarElementProps>;
+        numberOfMonths?: number;
+        onBlur?(e: React.FocusEvent<HTMLDivElement>): void;
+        onCaptionClick?(month: Date, e: React.MouseEvent<HTMLDivElement>): void;
+        onDayClick?(day: Date, modifiers: DayModifiers, e: React.MouseEvent<HTMLDivElement>): void;
+        onDayKeyDown?(day: Date, modifiers: DayModifiers, e: React.KeyboardEvent<HTMLDivElement>): void;
+        onDayMouseEnter?(day: Date, modifiers: DayModifiers, e: React.MouseEvent<HTMLDivElement>): void;
+        onDayMouseLeave?(day: Date, modifiers: DayModifiers, e: React.MouseEvent<HTMLDivElement>): void;
+        onDayTouchEnd?(day: Date, modifiers: DayModifiers, e: React.TouchEvent<HTMLDivElement>): void;
+        onDayTouchStart?(day: Date, modifiers: DayModifiers, e: React.TouchEvent<HTMLDivElement>): void;
+        onFocus?(e: React.FocusEvent<HTMLDivElement>): void;
+        onKeyDown?(e: React.KeyboardEvent<HTMLDivElement>): void;
+        onMonthChange?(month: Date): void;
+        pagedNavigation?: boolean;
+        renderDay?(date: Date, modifiers: Modifiers): React.ReactNode;
+        reverseMonths?: boolean;
+        selectedDays?: Modifier | Modifier[];
+        toMonth?: Date;
+        weekdayElement?: React.ReactElement<Partial<WeekdayElementProps>> |
+            React.ComponentClass<WeekdayElementProps> |
+            React.SFC<WeekdayElementProps>;
+        weekdaysLong?: [string, string, string, string, string, string, string];
+        weekdaysShort?: [string, string, string, string, string, string, string];
```

Comment:
real game changing feature would be to include `/** docs */` on each of these props. i know you have great docs for every prop already... but that could easily be added in a future release.